### PR TITLE
support NAS site

### PR DESCRIPTION
**Pull request by weiyuan-jiang**
Add support to NAS site

### CODE REVIEW

File: src/Applications/LDAS_App/ldas_setup [hunk 1]
```diff
@@ -1250,8 +1250,11 @@ class LDASsetup:
         fout =open(self.rundir+'/ldas_batchrun.j','w')
         fout.write("#!/bin/bash -f\n")
         jobid = None
+        SBATCHQSUB = 'sbatch'
         expid = self.rqdExeInp['EXP_ID']
-        fout.write("\nsed -i 's/if($capdate<$enddate) sbatch /#if($capdate<$enddate) sbatch /g' lenkf.j\n\n")
+        if self.GEOS_SITE == 'NAS':
+          SBATCHQSUB = 'qsub'
+        fout.write("\nsed -i 's/if($capdate<$enddate) "//SBATCHQSUB//" /#if($capdate<$enddate) qsub /g' lenkf.j\n\n")
```
Comment (gmao-rreichle) [inline src/Applications/LDAS_App/ldas_setup:1257]: Based on LDAS test failures on Discover for both myself and @biljanaorescanin, I suspect this line may not work as it stands.

**Comment by gmao-rreichle** [comment]
I tested the PR successfully on NCCS/Discover for LDAS_CONUS, LDAS_GLOBAL, LDAS_DEBUGCONUS, and LDAS_GNUCONUS, which is a sufficient subset of tests for this PR. 
@weiyuan-jiang : Please note on the PR when you had a chance to verify that GEOSldas runs at NAS.

**Comment by gmao-rreichle** [comment]
@weiyuan-jiang : I just noticed the merge conflict for this branch.  Please resolve it when you get a chance.  Thanks!

**Comment by weiyuan-jiang** [comment]
> @weiyuan-jiang : I just noticed the merge conflict for this branch. Please resolve it when you get a chance. Thanks!

resolved.

**Comment by weiyuan-jiang** [comment]
@gmao-rreichle  It runs on NAS

**Review by gmao-rreichle — APPROVED** [review]
I successfully ran the LDAS_CONUS test on Discover.

**Pull request merged**